docs/tutorials/multinode/terraform/init.sh

```diff
@@ -0,0 +1,28 @@
+# ---- Edit below -----#
+
+export TF_VAR_project=quantum-htcondor-15
```

Comment:
I would add comments to explain what all of these are and what the valid values are.

Reply:
My recommendation: for any of these vars that _must_ be replaced, have the default be invalid so the user doesn't e.g. get permission errors for working on the wrong project.